packages/wonder-blocks-dropdown/__snapshots__/generated-snapshot.test.js.snap

```diff
@@ -3615,7 +3615,7 @@ exports[`wonder-blocks-dropdown example 24 1`] = `
           "borderStyle": "solid",
           "borderWidth": 1,
           "boxSizing": "border-box",
-          "color": "rgba(33,36,44,0.64)",
+          "color": "#21242c",
```

Comment:
Why did the color change here?

Reply:
I think that it's because I changed that label to use `LabelMedium`.